EnglishFilter/sections/foreign.txt

```diff
@@ -6459,11 +6459,36 @@ zoomit.ir#$#body[style^="padding-top:"] { padding-top: 0!important; }
 zoomit.ir#$#body > .hotMenu[style="top: auto;"] { top: 0!important; }
 zoomit.ir#$#body > nav.is-sticky[style="top: 130px;"] { top: 0!important; }
 zoomit.ir#%#document.cookie = "BnShowNum=4; path=/;";
-soft98.ir#$#[id^="sidebar"][id*="stick"] > section { opacity: 0 !important; pointer-events: none !important; }
+soft98.ir#$#header[id]:has(.row) { remove: true; }
```

Comment:
Use syntax `example.com#$?#.ad { remove: true; }` for such rules.
Btw, they cause adb detection.

These rules work fine on my end. A directive is needed to limit platforms.
```
!#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android)
||soft98.ir/engine/modules/antibot/antibot.php
||soft98.ir/engine/ajax/controller.php
||file.soft98.ir^$image,redirect=nooptext
||cdn.hostdl.com^$image,redirect=nooptext
||linkdoni.soft98.ir^
||iranicard.ir^$domain=soft98.ir
||asiatech.ir^$domain=soft98.ir
||sysban.ir^$domain=soft98.ir
||smostafa.ir^$domain=soft98.ir
soft98.ir#$#.show.fade.tooltip { display: none !important; }
!#endif
```

Reply:
> they cause adb detection.

This site changes fast, if AdGuard merge filters slowly they'll become useless.